webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -6358,7 +6358,7 @@ var CURRENT_GROUP_DATA={};
                 }
             };
             _this.getScriptMessagePreview = function(message,emoji_template){
-				if (message && message.message && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.LOCATION && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.TEXT_HTML) {
+				if (message && message.message && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.LOCATION && message.contentType !== KommunicateConstants.MESSAGE_CONTENT_TYPE.TEXT_HTML && !Kommunicate.isRichTextMessage(message.metadata)) {
```

Comment:
@jithinbalu are you sure that `message.metadata` will always be there?

Reply:
Kommunicate.isRichTextMessage has a null check for metadata.